mne/baseline.py

```diff
@@ -24,7 +24,7 @@ def _log_rescale(baseline, mode='mean'):
 
 
 @verbose
-def rescale(data, times, baseline, mode='mean', copy=True, verbose=None):
+def rescale(data, times, baseline, mode='mean', copy=None, verbose=None):
```

Comment:
This is an API change. Now it will operate in place without warning. Not sure if this even should be public.